README.md

```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
```

Comment:
This needs to precede any other `cask` lines

Reply:
This is interesting as I’ve never seen or needed this before. I’ll add it back.

Reply:
Thanks!